src/app/core/services/fss-search-filter.service.ts

```diff
@@ -77,4 +77,20 @@ export class FssSearchFilterService {
     return operatorType
   }
 
+  getFilterExpressionForSimplifiedSearch(fssSearchFilter: string) {
+    let searchKeywords = fssSearchFilter.split(" ");
+     
+    var filterExpression = "";
```

Comment:
Please prefer `let` and `const` variable declarations over `var` declarations

Reply:
Resolved